bin/anthology/volumes.py

```diff
@@ -63,46 +86,44 @@ def _set_meta_info(self):
             month = month_str2num(self.get("month"))
             if month is not None:
                 self.attrib["meta_date"] = "{}/{}".format(self.get("year"), month)
-        if is_journal(self.top_level_id):
+        if is_journal(self.collection_id):
+            print('** ATTRIB', self.attrib)
```

Comment:
You can always use `log.debug()` for these things, then you don't need to remember to take them out later ;)